src/frontend/ML.ml

```diff
@@ -5,7 +5,8 @@ type 'a info =
   {con : 'a;
    span : Log.location}
 
-module Env = PersistentTable.M
+module T = PersistentTable.M
+type mlconf = {base_dir : string; indent : string}
```

Comment:
what is the `indent` in here do?

Reply:
Now I understand!! 

My suggestion is to change this to `import_depth : int`, and generate the appropriate indent-string when we're printing.

Reply:
@jonsterling I just thought it would be stupid to regenerate the string all the time. What would be the immortal API?

Reply:
@favonia I think it's ok to regenerate the string at all times, for the sake of modularity.